tools/cockpit.spec

```diff
@@ -444,12 +429,36 @@ fi
 %systemd_preun cockpit.socket cockpit.service
 
 %postun ws
-if [ -x %{_sbindir}/selinuxenabled ]; then
-    %selinux_modules_uninstall -s %{selinuxtype} %{name}
-    %selinux_relabel_post -s %{selinuxtype}
-fi
 %systemd_postun_with_restart cockpit.socket cockpit.service
 
+%package ws-selinux
+Summary: SELinux security policy for cockpit-ws
+# older -ws contained the SELinux policy, now split out
+Conflicts: %{name}-ws < 337-1.2025
```

Comment:
Please drop the `.2025`, as this will cause problems with real deployments.

Reply:
@Conan-Kudo what would break? If I drop it, that will break upgrades from 337-1 in Fedora/RHEL. It needs to be higher than 337-1 and lower than 337-1.20250425.., which isn't a lot of room. We could only make it "<= 337-1".

Reply:
This would be in 338, right? so it should be `338-1`.